R/tt_toString.R

```diff
@@ -66,13 +62,15 @@ setMethod("toString", "VTableTree", function(x,
 
 #' Table shells
 #'
-#' A table shell is a rendering of the table which maintains the structure, but does not
-#' display the values, rather displaying the formatting instructions for each cell.
+#' A table shell is a rendering of the table which maintains the structure, but does not display the values, rather
+#' displaying the formatting instructions for each cell.
 #'
-#' @inheritParams tostring
+#' @inheritParams formatters::toString
 #' @inheritParams gen_args
-#' @return for `table_shell_str` the string representing the table shell, for `table_shell`,
-#' `NULL`, as the function is called for the side effect of printing the shell to the console.
+#'
+#' @return
+#' * `table_shell` returns `NULL`, as the function is called for the side effect of printing the shell to the console.
```

Comment:
I dont get this explanation

Reply:
I believe it was meant that output will be printed to the console whenever the function is used, but if you assign `x <- table_shell(tbl)` then `x` will be `NULL`, which is not very useful.

Reply:
mmh yep it is a bit confusing. Anyway, I have never seen anyone using this, so we can see to it once there is a misunderstanding